sinks_sentry.go

```diff
@@ -12,10 +13,8 @@ import (
 // complete with stacktrace data and any additional context logged in the corresponding
 // log message (including anything accumulated on a sub-logger).
 type SentrySink struct {
-	// DSN configures the Sentry reporting destination
-	DSN string
-
-	options sentry.ClientOptions
+	// ClientOptions expose various options to configure the Sentry client
+	sentry.ClientOptions
```

Comment:
@jhchabran is this safe to make public? Why was it private to begin with? (just in case e.g. this struct is not read-only)

Reply:
Yes it is safe, it was unexported because we were passing the options through the constructor, which was enough. The current approach that @burmudar took gives more flexibility and enables to simply remove the additional constructor.